### PR TITLE
Vegitron/photo list

### PR DESCRIPTION
**Pull request by vegitron**

### CODE REVIEW

File: myuw/dao/enrollment.py [hunk 1]
```diff
@@ -9,6 +9,18 @@
...
+CLASS_CODES = {
+    "FRESHMAN": 1,
+    "SOPHMORE": 2,
+    "JUNIOR": 3,
+    "SENIOR": 4,
+    "GRADUATE": 5,
+}
+
+
+DEFAULT_CLASS_CODE = 6
```
Comment (fanglinfang) [inline myuw/dao/enrollment.py]: What 6 stands for? Are these codes used in SDB?

File: myuw/views/api/instructor_schedule.py [hunk 2]
```diff
@@ -322,3 +326,120 @@ def GET(self, request, year, quarter, curriculum,
...
+        timer = Timer()
+        try:
+            return self.make_http_resp(timer, year, quarter, curriculum,
+                                       course_number, course_section,
+                                       request)
+        except Exception:
+            return handle_exception(logger, timer, traceback)
+
+
+def not_instructor_error():
```
Comment (fanglinfang) [inline myuw/views/api/instructor_schedule.py]: please move this into rest_dispatch.py

File: myuw/dao/enrollment.py (hunk 1 above)
Comment (vegitron) [inline myuw/dao/enrollment.py]: Those are just numerical values we can use for sorting.  6 is just bigger than the highest value in the dictionary above.  If we add more defined sort orders, that value would need to change.  I could change it to len(CLASS_CODES.keys()+1, if that would help

File: myuw/views/api/instructor_schedule.py (hunk 2 above)
Comment (vegitron) [inline myuw/views/api/instructor_schedule.py]: 👍

**Pull request merged**